### PR TITLE
Google Trust Store Change Proposal

### PR DESCRIPTION
**Pull request by weirdscience**
Additional requirements from Google Trust Store Program minus the CT requirements.

### CODE REVIEW

File: certificate-policy.md [hunk 1]
```diff
@@ -654,6 +660,10 @@ Certificate applications SHALL be processed and a certificate issued within 30 d
 ### 4.3.1 CA actions during certificate issuance
 Certificate issuance by the Root CA SHALL require an individual authorized by the CA (i.e. the CA system operator, system officer, or PKI administrator) to deliberately issue a direct command in order for the Root CA to perform a certificate signing operation.
 
+All end entity certificates asserting the id-kp-serverAuth EKU SHALL assert a Certificate Transparency (CT) Signed Certificate Timestamp (SCT) via a certificate extension. At a minimum, the CA SHALL request the following number of SCTs:
+- One embedded SCT from a Google CT log
+- One embedded SCT from non-Google CT log
```
Comment (konklone) [inline certificate-policy.md:665]: :+1: But it's worth noting that by the time anything is in operation and undergoing an audit, Mozilla may have finalized a new CT policy, and/or Google may have changed theirs in tandem with their intended CT-for-everything 2017 policy.

File: certificate-policy.md [hunk 2]
```diff
@@ -1088,6 +1089,12 @@ The business continuity plan MUST include:
 14. The distance of recovery facilities to the CA's main site; and
 15. Procedures for securing its facility to the extent possible during the period of time following a disaster and prior to restoring a secure environment either at the original or a remote site.
 
+In the event of a mis-issuance, the issuing CA SHALL conduct the following actions:
+1. Communicate with the FPKIPA and any parties that might be affected of the mis-issuance;
+2. Revoke any mis-issued certificates;
+3. Publish a notice of the revocation on a publicly accessible website; and
+4. Conduct a full post-mortem and publicly publish the findings on a publicly accessible website.
```
Comment (konklone) [inline certificate-policy.md:1106]: :100: :100: :100:

**Comment by konklone** [comment]
This PR is to the `CAB-NetworkSecReqs` branch, not `master`. After resolution, you may want to close and re-open the agreed-upon changes to the `master` branch instead of merging this one.

File: certificate-policy.md (hunk 1 above)
Comment (weirdscience) [reply certificate-policy.md:665]: Yep, moving target and this assumes that Fed TLS certs have a validity of less than 15 months.

**Pull request merged**